classes/class-p4-sitemap.php

```diff
@@ -112,6 +112,7 @@ public function get_evergreen_pages() : array {
 				'meta_key'         => '_wp_page_template',
 				'meta_value'       => 'page-templates/evergreen.php',
 				'suppress_filters' => false,
+				'numberposts'      => -1,
```

Comment:
It's good to be an upper limit on this (maybe 100 posts) just in case. You can use a constant for this like 
https://github.com/greenpeace/planet4-plugin-blocks/blob/develop/classes/controller/blocks/class-articles-controller.php#L32

Or even better retrieve only the fields needed for this (id, permalink) using fields query argument.
https://developer.wordpress.org/reference/classes/wp_query/parse_query/.
I know this is out of scope :) just a suggestion

Reply:
Added the limit, as for the `fields` part, I tried adding a `fields` key to `args` but it didn't change anything, so I left the current query as is, as I assumed that will need refactoring the current code into using something different than `get_posts`. 🤔

Reply:
No problem, we can revisit later.
Thank you! :+1: